Makefile

```diff
@@ -32,7 +32,7 @@ lint:
 
 .PHONY: gen-proto
 gen-proto:
-	${GOBIN}/buf generate --template buf.gen.yaml
```

Comment:
The `--template buf.gen.yaml` flag doesn't seem present in the new script. Is it in fact getting used? Or does the script work without it?

Reply:
Yep! `buf generate` will use `buf.gen.yaml` by default if no template is provided. I forgot to remove the `buf.gen.yaml` template on the [previous PR](https://github.com/polymerdao/monomer/pull/115#discussion_r1710209394) after removing pulsar and renaming it.